public/app/themes/clarity/inc/admin/users/add-agency-admin.php

```diff
@@ -8,6 +8,10 @@
  * Access to everything within that agency minus a few Administrator settings.
  *
  * Type of user: heads of agencies, non-developer intranet team members
+ * 
+ * If you edit this file, you must sync the roles to the database by running the 
+ * SyncUserRoles admin action. 
+ * Navigate to Tools > Admin Commands > Sync user roles from codebase to database
```

Comment:
Can we create a shell script to execute when the container starts for the first time, instead of relying on a manual action?

```shell
wp run my-custom-command
```

Reply:
Great shout 🙌